resources/metadata/malwareScan.yaml

```diff
@@ -0,0 +1,44 @@
+metadata:
+  name: malwareScan
+  description: Performs a malware scan
+  longDescription: |
+    Performs a malware scan
+spec:
+  inputs:
```

Comment:
@radsoulbeard 

Do you think we need more parameters, eg. don't fail in case encrypted content is detected (... in case we need a malwarescan in the narrower sense), don't fail in case sha256 hash does not match (could be a security issue ...)

As always: the more parameters, the more coding ... Right how we have the minimal set of parameters.

Reply:
I would say, let's start small.